### PR TITLE
chore(package): update prettier to version 1.4.2

### PR DESCRIPTION
**Pull request by hawkrives**
Closes #1132

### CODE REVIEW

File: source/lib/courses/load-all-courses.js [hunk 1]
```diff
@@ -6,9 +6,7 @@
 
 import type {CourseCollectionType} from './types'
 import {loadStudentNumber} from './load-student-number'
-import {
-  loadCoursesFromUnofficialTranscript,
-} from './load-courses-from-unofficial-transcript'
+import {loadCoursesFromUnofficialTranscript} from './load-courses-from-unofficial-transcript'
```
Comment (hawkrives) [inline source/lib/courses/load-all-courses.js:9]: yay

File: source/storybook/index.js [hunk 2]
```diff
@@ -53,8 +53,7 @@ import SettingsView from '../views/settings'
 import CreditsView from '../views/settings/credits'
 import PrivacyView from '../views/settings/privacy'
 import LegalView from '../views/settings/legal'
-import CredentialsLoginSection
-  from '../views/settings/sections/login-credentials'
+import CredentialsLoginSection from '../views/settings/sections/login-credentials'
```
Comment (hawkrives) [inline source/storybook/index.js:56]: yay

File: source/storybook/index.js [hunk 3]
```diff
@@ -74,11 +73,10 @@ type ViewCollectionType = {
   },
 }
 
-const Nav = ({children}: {children?: Function}) => (
+const Nav = ({children}: {children?: Function}) =>
```
Comment (hawkrives) [inline source/storybook/index.js:76]: hrm

File: source/views/building-hours/detail.android.js [hunk 4]
```diff
@@ -150,29 +150,29 @@ export class BuildingHoursDetailView extends React.Component {
...
                   <ScheduleRow
                     key={i}
                     now={this.state.now}
                     schedule={schedule}
                     isActive={
                       set.isPhysicallyOpen !== false &&
-                        schedule.days.includes(dayOfWeek) &&
-                        isBuildingOpenAtMoment(schedule, this.state.now)
+                      schedule.days.includes(dayOfWeek) &&
+                      isBuildingOpenAtMoment(schedule, this.state.now)
```
Comment (hawkrives) [inline source/views/building-hours/detail.android.js:169]: grumble. I liked the indented boolean expressions.

File: source/views/menus/components/filter-menu-toolbar.js [hunk 5]
```diff
@@ -42,7 +42,9 @@ export function FilterMenuToolbar({date, title, filters, onPress}: PropsType) {
         isActive={isFiltered}
         title={
           isFiltered
-            ? `${appliedFilterCount} ${appliedFilterCount === 1 ? 'Filter' : 'Filters'}`
+            ? `${appliedFilterCount} ${appliedFilterCount === 1
+                ? 'Filter'
+                : 'Filters'}`
```
Comment (hawkrives) [inline source/views/menus/components/filter-menu-toolbar.js]: I think we'll want to pull this out into its own variable sometime.

File: source/views/transportation/bus/index.js [hunk 6]
```diff
@@ -48,7 +48,8 @@ export default class BusView extends React.Component {
     if (!activeBusLine) {
       return (
         <NoticeView
-          text={`The line "${this.props.line}" was not found among ${busLines
+          text={`The line "${this.props
+            .line}" was not found among ${busLines
             .map(({line}) => line)
             .join(', ')}`}
```
Comment (hawkrives) [inline source/views/transportation/bus/index.js]: this whole string needs to be restructured into some variables

File: source/views/transportation/otherModes.js [hunk 7]
```diff
@@ -46,12 +46,14 @@ export default function OtherModesView() {
             onPress={() =>
               trackedOpenUrl({
                 url: data.url,
-                id: `Transportation_OtherModes_${data.name.replace(' ', '')}View`,
+                id: `Transportation_OtherModes_${data.name.replace(
+                  ' ',
+                  '',
+                )}View`,
```
Comment (hawkrives) [inline source/views/transportation/otherModes.js]: extract this into a variable

**Review by hawkrives — COMMENTED** [review]
a mixed bag, for the first time.

**Review by drewvolz — APPROVED** [review]

**Pull request merged**